config/laratrust.php

```diff
@@ -270,6 +270,17 @@
         */
         'register' => false,
 
+        /*
+        |--------------------------------------------------------------------------
+        | Laratrust Panel Domain
+        |--------------------------------------------------------------------------
+        |
+        | This is the Domain Laratrust panel for roles and permissions
+        | will be accessible from.
+        |
+        */
+        'domain' => env('LARATRUST_PANEL_DOMAIN', env('LARATRUST_PANEL_DOMAIN', (app()->runningInConsole() === false) ? request()->getHost() : 'localhost')),
```

Comment:
This is interesting and I'm excited to play around with it

Reply:
I have not messed with this feature, but it does look neat. Here is the readme: https://laratrust.santigarcor.me/docs/8.x/usage/admin-panel.html#how-to-customize-the-views

Reply:
That looks like it can make a bunch of stuff easier for us which is nice. It is locked down to only admin correct?

Reply:
@iccowan Panel is disabled by default, see config/laratrust.php:
`'panel' => [
        /*
        |--------------------------------------------------------------------------
        | Laratrust Panel Register
        |--------------------------------------------------------------------------
        |
        | This manages if routes used for the admin panel should be registered.
        | Turn this value to false if you don't want to use Laratrust admin panel
        |
        */
        'register' => false,`
Let's leave this disabled for now. We can create another story to enable this and place it behind perimeter security in the future.